src/components/inline-inputs/index.ts

```diff
@@ -1,2 +1,2 @@
-export { default, InlineInputsContext } from "./inline-inputs.component";
+export { default } from "./inline-inputs.component";
```

Comment:
this is an internal util, it was not documented anywhere, and is only used to pass the label from inline input, so I can't see how it could be a breaking change.

Reply:
OK, probably fair enough if it's not documented - just that consumers could in theory be importing it for some reason (no idea why but it was possible) and that would now break. But it's probably OK to not mention it, especially as this PR will already be marked as a breaking change.